### PR TITLE
Add Eventing Kafka WG to community section

### PR DESCRIPTION
**Pull request by devguyio**
# Changes

- Add Eventing Kafka WG charter
- Add Eventing Kafka WG entry in the WORKING-GROUPS.md

Related to: #693

### CODE REVIEW

File: working-groups/WORKING-GROUPS.md [hunk 1]
```diff
@@ -192,6 +193,27 @@ Event delivery data plane.
...
+| Slack Channel              | [#eventing-kafka](https://slack.knative.dev/messages/eventing-kafka)                                                     |
+| Github Team WG leads       | [@knative/kafka-wg-leads](https://github.com/orgs/knative/teams/kafka-wg-leads/members)                                                                       |
+
+| &nbsp;                                                           | Leads           | Company | Profile                                           |
+| ---------------------------------------------------------------- | --------------- | ------- | ------------------------------------------------- |
+| <img width="30px" src="https://github.com/devguyio.png">         | Ahmed Abdalla   | Red Hat | [devguyio](https://github.com/devguyio)           |
+| <img width="30px" src="https://github.com/lionelvillard.png">    | Lionel Villard  | IBM     | [lionelvillard](https://github.com/lionelvillard) |
+| <img width="30px" src="https://github.com/travis-minke-sap.png"> | Travis Minke    | SAP     | [csantanapr](https://github.com/travis-minke-sap) |
+
```
Comment (travis-minke-sap) [inline working-groups/WORKING-GROUPS.md:216]: not sure what "csantanapr" is but looks like it should be "travis-minke-sap" ?

File: working-groups/WORKING-GROUPS.md [hunk 2]
```diff
@@ -192,6 +193,27 @@ Event delivery data plane.
 | -------------------------------------------------------- | ------------------- | ------- | ------------------------------------------------------------------ |
 | <img width="30px" src="https://github.com/matzew.png">   | Matthias Wessendorf | Red Hat | [matzew](https://github.com/matzew)                                |
 
+## Eventing Kafka
+
+A dedidacetd working group for Kafka based Knative Eventing components.
+
```
Comment (travis-minke-sap) [inline working-groups/WORKING-GROUPS.md:199]: spelling of "dedicated"

File: working-groups/WORKING-GROUPS.md (hunk 1 above)
Comment (lionelvillard) [reply working-groups/WORKING-GROUPS.md:216]: Carlos Santana!

File: working-groups/kafka/CHARTER.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# Motivation/Abstract
+
+Within the knative-sandbox org, Kafka based eventing currently includes 2 channel implementations, 2 sources implementations, 1 broker and 1 sink with more than three vendors and multiple users for production workloads. With that, it is desired to have a dedicated forum for Kafka based Eventing which can be really beneficial both for exposure and focus.
+
+This is a proposal to create a new working group that would be a sub working group of Eventing and would report a brief status summary to the Eventing WG being a sub working group.
+
+# Mission Statement
+
+The Eventing Kafka working group is the innovation forum for Kafka experts, end users and vendors wanting to leverage Kafka for Knative Eventing components. The working group enables a diversity of Kafka based components while ensuring Eventing users end up with a smooth UX for choosing, running and maintaining those components and that Eventing Kafka contributors are algined on a set of working group standards.
+
```
Comment (lionelvillard) [inline working-groups/kafka/CHARTER.md:12]: algined -> aligned

File: working-groups/kafka/CHARTER.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+# Goals
+
+1. Align the future work across all Kafka based eventing components into a single Kafka focused roadmap.
+2. Promote eventing Kafka to the community and attract more Kafka end users, adopters, and contributors.
+3. Narrow-focus on Kafka features, Kafka docs and Kafka UX and contribEx.
```
Comment (lionelvillard) [inline working-groups/kafka/CHARTER.md:18]: contribEx? maybe contributor onboarding?

File: working-groups/kafka/CHARTER.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+1. Align the future work across all Kafka based eventing components into a single Kafka focused roadmap.
+2. Promote eventing Kafka to the community and attract more Kafka end users, adopters, and contributors.
+3. Narrow-focus on Kafka features, Kafka docs and Kafka UX and contribEx. 
+
+# In Scope
+
+* Maintain a roadmap that covers all the Kafka based projects within knative-sandbox.
+* Funnel the feedback of Kafka eventing components and surface that to other working groups (e.g. to unblock a specific feature development or enhance the user expeience of Kafka based Eventing components)
+* Work with the Docs WG to maintain and enhance the Kafka components docs.
```
Comment (lionelvillard) [inline working-groups/kafka/CHARTER.md:24]: expeience -> experience

File: working-groups/WORKING-GROUPS.md (hunk 1 above)
Comment (devguyio) [reply working-groups/WORKING-GROUPS.md:216]: My copy paste skills are horrible after 9 pm 😅 sorry lol

File: working-groups/kafka/CHARTER.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+# Kafka Eventing Working Group
+
+# Motivation/Abstract
+
+Within the knative-sandbox org, Kafka based eventing currently includes 2 channel implementations, 2 sources implementations, 1 broker and 1 sink with more than three vendors and multiple users for production workloads. With that, it is desired to have a dedicated forum for Kafka based Eventing which can be really beneficial both for exposure and focus.
+
+This is a proposal to create a new working group that would be a sub working group of Eventing and would report a brief status summary to the Eventing WG being a sub working group.
```
Comment (rhuss) [inline working-groups/kafka/CHARTER.md:7]: If merged, it's not a proposal anymore. You should describe the mission as a given here or just skip this sentence as there is a mission statement already below.

**Review by rhuss — COMMENTED** [review]
minor nit, otherwise looks good to me.

**Comment by evankanderson** [comment]
Can you also update at least `peribolos/knative-sandbox.yaml` and `peribolos/knative.yaml`? Bonus points for running `./hack/update-codegen.sh`.

**Comment by devguyio** [comment]
@rhuss @evankanderson done, also run `update-codegen.sh`.

**Comment by devguyio** [comment]
```
 failed to list 0(Eventing Kafka Writers) members: return code not 2XX: 404 Not Found",
```
@evankanderson does this need to be created manually by somebody first?

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

It looks like we haven't run the autogen in a little while; I have a todo to automate it. ☹️

I think the periobolos failures are expected; it tries to create the group earlier in the run, but isn't clever enough to skip past the 404 later.

/assign @pmorie 

We'll need steering for the OWNERS_ALIASES file.

File: knative-sandbox-OWNERS_ALIASES [hunk 7]
```diff
@@ -183,21 +190,26 @@ aliases:
   - zhanggbj
   net-certmanager-approvers:
   - ZhiminXiang
+  - tcnghia
```
Comment (pmorie) [inline knative-sandbox-OWNERS_ALIASES:193]: was this intentional?

Comment (devguyio) [reply knative-sandbox-OWNERS_ALIASES:193]: that's a result of running `update-codegen.sh` which hasn't run for a while after other changes to peribolos were merged. This one in particular was introduced [here](https://github.com/knative/community/commit/f0ee0738a1656dd3a9f4002a9e05ff34cac25266#diff-48ae180ea1cda2986f85c0177aae1f359defa7eb1f879a0789bb4ebaf85e3faaR216)

File: OWNERS_ALIASES [hunk 8]
```diff
@@ -28,14 +26,17 @@ aliases:
...
   - rhuss
-  conformance-wg-leads:
-  - tayarani
+  conformance-task-force-leads:
+  - omerbensaadon
+  - salaboy
   conformance-writers:
-  - tayarani
+  - omerbensaadon
+  - salaboy
```
Comment (devguyio) [inline OWNERS_ALIASES:34]: @pmorie these were introduced [here](https://github.com/knative/community/pull/763)

File: OWNERS_ALIASES [hunk 9]
```diff
@@ -28,14 +26,17 @@ aliases:
...
   docs-reviewers:
   - RichardJJG
   - jonatasbaldin
   - mpetason
+  - nainaz
```
Comment (devguyio) [inline OWNERS_ALIASES:39]: @pmorie and this was introduced [here](https://github.com/knative/community/pull/765)

**Comment by evankanderson** [comment]
Yeah, the changes in OWNER_ALIASES are due to my still trying to get "some but not all" codegen jobs running in community...

**Pull request merged**